app/assets/stylesheets/users/profile/donations_by_causes.scss

```diff
@@ -0,0 +1,42 @@
+// All these are pretty much just place holders. Still playing around with them. 
+// bar sizes aren't really consistent yet. need to figure that out. 
```

Comment:
you don't really need to commit these comment messages. Unless you want to put `TODO` here

Reply:
Yeah, I can put a TODO. I just have the comments in there mostly because this pr doesn't require complete implementation of that section.

Reply:
Yeah that's fair but that's more or less understood as this story was design implementation with no data attached